config-templates/ridge/ridge-csi-driver/1.0.0/storage-class-parameters.json

```diff
@@ -12,6 +12,19 @@
 		"max-length": 64,
 		"category": "config"
 	},
+	{
```

Comment:
General recommendation is, whenever we want to add a new parameter to the template. We have to create a new version of the template.

Reply:
It definitely makes sense, but in this particular situation it's completely backwards compatible: "no param" means the "default storage class" — the same one used before our backed started to support multiple storage classes.

Is there a reason to release a new version in this case?